.github/workflows/test.yml

```diff
@@ -12,13 +12,32 @@ jobs:
     strategy:
       matrix:
         include:
-        - { python-version: "3.7", os: "ubuntu-latest" }
-        - { python-version: "3.8", os: "ubuntu-latest" }
-        - { python-version: "3.9", os: "ubuntu-latest" }
+        - { python-version: "3.7", os: "ubuntu-latest", backend-db: sqlite }
+        - { python-version: "3.8", os: "ubuntu-latest", backend-db: sqlite }
+        - { python-version: "3.9", os: "ubuntu-latest", backend-db: sqlite }
+        - { python-version: "3.7", os: "ubuntu-latest", backend-db: postgresql }
+        - { python-version: "3.8", os: "ubuntu-latest", backend-db: postgresql }
+        - { python-version: "3.9", os: "ubuntu-latest", backend-db: postgresql }
       fail-fast: false
 
-    name: Pytest on py${{ matrix.python-version }} (${{ matrix.os }})
+    name: "Pytest on py${{ matrix.python-version }} (OS: ${{ matrix.os }}, DB: ${{ matrix.backend-db }})"
     runs-on: ${{ matrix.os }}
+    services:
+      postgres:
+        # Docker Hub image
+        image: postgres:11
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: postgres
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          # Maps tcp port 5432 on service container to the host
+          - 5432:5432
```

Comment:
Starting Postgres even for SQLite tests is wasteful, but there doesn't seem to be any way around that: https://github.community/t/conditional-services-in-a-job/135301

Reply:
Could you please add a comment between `services:` and `postgres:` explaining this?